### ISSUE TITLE
AUR broken packages vs working cloning source and execution

### ISSUE THREAD

**Issue by Elia-Sh** [evolus/pencil#697]
Hi, 
  TL;DR I suspect that the AUR packages are  broken, 
  after installation of pencil from AUR, pencil will **hang with a spinning logo**,
  probably need to update `PKGBUILD` in aur..

  No error messages during execution  - 
  ```
  $ pencil 
  useHWAConfig:  false
  Hardware acceleration disabled for Linux.
  Background web-printer started.
  Shortcut main service started.
  (node:88769) ProtocolDeprecateCallback: The callback argument of protocol module APIs is no longer needed.
  (Use `electron --trace-warnings ...` to show where the warning was created)
  ```
  
  
  * Operating system:
  ```
  Operating System: Arch Linux                       
            Kernel: Linux 5.13.6-arch1-1
      Architecture: x86-64
  ```
  * Pencil version:
  
  `Electron v13.1.8 | Chromium v91.0.4472.164 | Node v14.16.0 | v8 v9.1.269.39-electron.0`
  
  
  install from AUR - https://aur.archlinux.org/packages/pencil/
  ```
  $ pacman -Q --info pencil
  Name            : pencil
  Version         : 3.1.0-7
  Description     : Sketching and GUI prototyping/wireframing tool
  Architecture    : any
  URL             : https://github.com/evolus/pencil
  Licenses        : GPL2
  Groups          : None
  Provides        : None
  Depends On      : electron
  Optional Deps   : None
  Required By     : None
  Optional For    : None
  Conflicts With  : evolus-pencil-bin  pencil-v2
  Replaces        : None
  Installed Size  : 48.32 MiB
  Packager        : Unknown Packager
  Build Date      : Tue 10 Aug 2021 10:43:38 PM IDT
  Install Date    : Wed 11 Aug 2021 09:11:39 AM IDT
  Install Reason  : Explicitly installed
  Install Script  : No
  Validated By    : None
  ```
  
  also tested - `evolus-pencil-git` - resulted in same execution with spinning logo.

  ```
  $ pacman -Q --info evolus-pencil-git
  Name            : evolus-pencil-git
  Version         : 1521.f1309bc-1
  Description     : An open-source GUI prototyping tool
  Architecture    : x86_64
  URL             : http://pencil.evolus.vn/
  Licenses        : GPL2
  Groups          : None
  Provides        : evolus-pencil-bin  evolus-pencil-avn
  Depends On      : electron
  Optional Deps   : None
  Required By     : None
  Optional For    : None
  Conflicts With  : evolus-pencil-bin  evolus-pencil-svn
  Replaces        : evolus-pencil-svn
  Installed Size  : 65.15 MiB
  Packager        : Unknown Packager
  Build Date      : Wed 11 Aug 2021 09:41:32 AM IDT
  Install Date    : Wed 11 Aug 2021 09:44:09 AM IDT
  Install Reason  : Explicitly installed
  Install Script  : No
  Validated By    : None
  ```

The awesome thing is that building the source from git, results in successful execution of the app.

```
$ npm start

> start
> ./node_modules/.bin/electron ./app

useHWAConfig:  false
Hardware acceleration disabled for Linux.
Background web-printer started.
Shortcut main service started.
(node:88934) ProtocolDeprecateCallback: The callback argument of protocol module APIs is no longer needed.
(node:88934) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
(node:88984) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Shortcut: Super+F12 registered
RENDERER started.
(node:88934) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
```

Starting the pencil-cloned-from-git-app with global `electron` works as well - 
```
 which electron
/usr/bin/electron
$ electron -v                                                                                                   
v13.1.8
$ pwd
/home/elia/git/pencil
$ electron ./app
useHWAConfig:  false
Hardware acceleration disabled for Linux.
Background web-printer started.
Shortcut main service started.
(node:89197) ProtocolDeprecateCallback: The callback argument of protocol module APIs is no longer needed.
(Use `electron --trace-warnings ...` to show where the warning was created)
(node:89245) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `exe --trace-deprecation ...` to show where the warning was created)
Shortcut: Super+F12 registered
RENDERER started.
```

The above does not work with the installed version from AUR - the app still hangs on the spinning logo.

**Comment by hoaivan** [comment]
You should use electron9. There was compatible issue with electron 13.

**Comment by Elia-Sh** [comment]
Hi @hoaivan, 

Could you please elaborate about the comparability issue between latest electron and the version used by pencil?

I was able to use pencil with electron 13 - latest available electron version on arch linux, and as described in my comment above,
did not see any issues with pencil.

in addition I've created an AUR package that follows the development branch on `evolus/pencil @ github` -
https://aur.archlinux.org/packages/evolus-pencil-git-dev-branch/

**Comment by Elia-Sh** [comment]
using the `development` tracking AUR package solves the spinning logo + hanged app on execution,
I've opened https://github.com/evolus/pencil/issues/699 to progress the master branch.

**Comment by hoaivan** [comment]
@Elia-Sh I tried electron13 on development branch. No more hang spinning logo but i found 2 other issues:
- Broken backgroud when i tried to set a page as background of others  
- Very slow saving to file

Switch back to electron9 on development branch solves all these!
